src/CallManager.cpp

```diff
@@ -490,6 +502,69 @@ CallManager::stopRingtone()
         player_.setPlaylist(nullptr);
 }
 
+QStringList
+CallManager::windowList()
+{
+        windows_.clear();
+        windows_.push_back({"Entire screen", 0});
```

Comment:
You probably want this translatable, i.e. add a `tr` or `Qt::translate` or so :3

Reply:
Thanks. Spot the native English speaker.

Reply:
Well, I have it in english as well, but people always complain abut missing translations... :D